### ISSUE TITLE
Extension enable make paste in other extension webview not work.

### ISSUE THREAD

**Issue by icewind7030** [gazugafan/vscode-indent-on-paste#28]
After VSCode updated, the paste behavior in code feels bad, I test many extensions to fix it and finally this extension makes it much better.
But one problem I met today is that, when I tried to paste some text in a vscode webview of a third party custom extension,  the paste aciton not work.
And I can't find any extension setting to fix my problem except disable it temporary.

The third party extension I use is called 'Alkaid' if you ask, thanks.

**Comment by gazugafan** [comment]
Sounds like #15 
Maybe use CTRL+SHIFT+V instead of CTRL+V when pasting outside the editor?

**Comment by icewind7030** [comment]
I tried to set keyboard shortcut like this without any other key "v" setting:
```
{
    "key": "ctrl+cmd+v",
    "command": "editor.action.clipboardPasteAction",
    "when": "editorTextF`ocus && !editorReadonly"
  }
```
But seems not work in my situation.

**Comment by gazugafan** [comment]
Looks like you have a typo in the "when" line. That would be cool if it works, though. Let us know!

**Comment by icewind7030** [comment]
Sorry for the mistake when paste my setting, I checked and there's no typo in my actual keyboard setting. 
And I also check another extension called 'GitLab Workflow' which has custom webview, too. My problem still the same😂.

**Comment by gazugafan** [comment]
Yeah, it's almost certainly the same as #15. I know it's not ideal, but CTRL+SHIFT+V should bypass this extension and do a raw paste. Does that work?

**Comment by icewind7030** [comment]
Sorry for the late response, my vs code keyboard shortcut setting is like blew: 
```json
// 将键绑定放入此文件中以覆盖默认值
[
  {
    "key": "ctrl+cmd+space",
    "command": "editor.action.triggerSuggest",
    "when": "editorHasCompletionItemProvider && editorTextFocus && !editorReadonly"
  },
  {
    "key": "ctrl+space",
    "command": "-editor.action.triggerSuggest",
    "when": "editorHasCompletionItemProvider && editorTextFocus && !editorReadonly"
  },
  {
    //重做
    "key": "cmd+y",
    "command": "redo",
    "when": "editorTextFocus && !editorReadonly"
  },
  {
    "key": "cmd+l",
    "command": "expandLineSelection"
  },
  {
    "key": "shift+cmd+b",
    "command": "-workbench.action.tasks.build"
  },
  {
    "key": "shift+cmd+b",
    "command": "HookyQR.beautify"
  },
  {
    "key": "ctrl+cmd+w",
    "command": ""
  },
  {
    "key": "shift+cmd+numpad8",
    "command": "workbench.action.terminal.focusPrevious"
  },
  {
    "key": "shift+cmd+numpad2",
    "command": "workbench.action.terminal.focusNext"
  },
  {
    "key": "cmd+shift+v",
    "command": "editor.action.clipboardPasteAction",
  }
]
```
And I am unable to paste content on third-party WebViews such as User Setting, regardless of whether I comment out the cmd+shift+v part of the code or not.

Can you figure out which part of my setting is wrong? I tried remove all the extensions may related with keybinds, still no luck.

**Comment by icewind7030** [comment]
I also tried to right click in the User Setting search input, which would popup a context menu with paste action inside, but still nothing happend. If I disabled `Indent and Paste` extension, it works.
<img width="717" alt="image" src="https://github.com/gazugafan/vscode-indent-on-paste/assets/7500226/5f1532f8-cbab-4d0a-b26d-4a298102f8e4">

**Comment by gazugafan** [comment]
I'm not sure. In my experience, this only happens when this extension's ```Paste Method``` setting is set to ```workaround``` (not ```native```). Although, I think there were some reports of it happening for some people when set to ```native``` as well.

Pressing CTRL+SHIFT+V bypasses the extension and gets around the problem. I didn't modify any of my keyboard shortcuts. I think this works on Mac as well.

So, you should be able to either use the ```native``` pasting method setting, or you can get around this by pressing CMD+SHIFT+V instead of CMD+V when pasting outside the editor.
